test/clt-tests/sharding/replication/replication-test.rec

```diff
@@ -0,0 +1,282 @@
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 4. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+mysql -h0 -P5306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 4. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+export INSTANCE=4
+––– output –––
+––– block: ../base/flexible/stop-searchd –––
+––– input –––
+sleep 5;mysql -h0 -P2306 -e "DELETE FROM replication:testrt WHERE id = 6775836565403860994"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P2306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+export INSTANCE=4
+––– output –––
+––– block: ../base/flexible/start-searchd-precach –––
+––– input –––
+sleep 5;mysql -h0 -P1306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+mysql -h0 -P3306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+mysql -h0 -P4306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
```

Comment:
I suggest to use ; as delimiter or grep in case we use \G, there is no point to use \G and ; together, they work differently in MySQL, choose the one